static/locales/en.client.json

```diff
@@ -1,779 +1,748 @@
   },
   "ViewLayoutMenu": {
-    "DeleteRecord": "Delete record",
-    "CopyAnchorLink": "Copy anchor link",
-    "ShowRawData": "Show raw data",
-    "PrintWidget": "Print widget",
-    "DownloadCSV": "Download as CSV",
-    "DownloadXLSX": "Download as XLSX",
-    "EditCardLayout": "Edit Card Layout",
-    "WidgetOptions": "Widget options",
-    "AdvancedSortFilter": "Advanced Sort & Filter",
-    "DataSelection": "Data selection",
-    "OpenConfiguration": "Open configuration",
-    "DeleteWidget": "Delete widget"
+    "Advanced Sort & Filter": "Advanced Sort & Filter",
+    "Copy anchor link": "Copy anchor link",
+    "Data selection": "Data selection",
+    "Delete record": "Delete record",
+    "Delete widget": "Delete widget",
+    "Download as CSV": "Download as CSV",
+    "Download as XLSX": "Download as XLSX",
+    "Edit Card Layout": "Edit Card Layout",
+    "Open configuration": "Open configuration",
+    "Print widget": "Print widget",
+    "Show raw data": "Show raw data",
+    "Widget options": "Widget options"
   },
   "ViewSectionMenu": {
-    "UpdateSortFilterSettings": "Update Sort&Filter settings",
-    "Save": "Save",
+    "(customized)": "(customized)",
+    "(empty)": "(empty)",
+    "(modified)": "(modified)",
+    "Custom options": "Custom options",
+    "FILTER": "FILTER",
     "Revert": "Revert",
-    "SortedBy": "Sorted by",
-    "AddFilter": "Add Filter",
-    "ToggleFilterBar": "Toggle Filter Bar",
-    "FilteredBy": "Filtered by",
-    "Customized":"(customized)",
-    "Modified":"(modified)",
-    "Empty":"(empty)",
-    "CustomOptions":"Custom options",
-    "Sort": "SORT",
-    "Filter": "FILTER"
-  },
-  "aclui": {
-    "AccessRules": {
-      "Checking": "Checking...",
-      "Saved": "Saved",
-      "Invalid": "Invalid",
-      "Save": "Save",
-      "Reset": "Reset",
-      "AddTableRules": "Add Table Rules",
-      "AddUserAttributes": "Add User Attributes",
-      "Users": "Users",
-      "UserAttributes": "User Attributes",
-      "AttributeToLookUp": "Attribute to Look Up",
-      "LookupTable": "Lookup Table",
-      "LookupColumn": "Lookup Column",
-      "DefaultRules": "Default Rules",
-      "Condition": "Condition",
-      "Permissions": "Permissions",
-      "RulesForTable": "Rules for table ",
-      "AddColumnRule": "Add Column Rule",
-      "AddDefaultRule": "Add Default Rule",
-      "DeleteTableRules": "Delete Table Rules",
-      "SpecialRules": "Special Rules",
-      "AccessRulesDescription": "Allow everyone to view Access Rules.",
-      "FullCopiesDescription": "Allow everyone to copy the entire document, or view it in full in fiddle mode.\nUseful for examples and templates, but not for sensitive data.",
-      "AccessRulesName": "Permission to view Access Rules",
-      "FullCopies": "Permission to access the document in full when needed",
-      "AttributeNamePlaceholder": "Attribute name",
-      "Everyone": "Everyone",
-      "EveryoneElse": "Everyone Else",
```

Comment:
This key is still used in ACL page.

Reply:
They are not really removed either. Here the new `AccessRules` translations keys:
```javascript
"AccessRules": {
    "Add Column Rule": "Add Column Rule",
    "Add Default Rule": "Add Default Rule",
    "Add Table Rules": "Add Table Rules",
    "Add User Attributes": "Add User Attributes",
    "Allow everyone to copy the entire document, or view it in full in fiddle mode.\nUseful for examples and templates, but not for sensitive data.": "Allow everyone to copy the entire document, or view it in full in fiddle mode.\nUseful for examples and templates, but not for sensitive data.",
    "Allow everyone to view Access Rules.": "Allow everyone to view Access Rules.",
    "Attribute name": "Attribute name",
    "Attribute to Look Up": "Attribute to Look Up",
    "Checking...": "Checking...",
    "Condition": "Condition",
    "Default Rules": "Default Rules",
    "Delete Table Rules": "Delete Table Rules",
    "Enter Condition": "Enter Condition",
    "Everyone": "Everyone",
    "Everyone Else": "Everyone Else",
    "Invalid": "Invalid",
    "Lookup Column": "Lookup Column",
    "Lookup Table": "Lookup Table",
    "Permission to access the document in full when needed": "Permission to access the document in full when needed",
    "Permission to view Access Rules": "Permission to view Access Rules",
    "Permissions": "Permissions",
    "Remove column {{- colId }} from {{- tableId }} rules": "Remove column {{- colId }} from {{- tableId }} rules",
    "Remove {{- tableId }} rules": "Remove {{- tableId }} rules",
    "Remove {{- name }} user attribute": "Remove {{- name }} user attribute",
    "Reset": "Reset",
    "Rules for table ": "Rules for table ",
    "Save": "Save",
    "Saved": "Saved",
    "Special Rules": "Special Rules",
    "Type a message...": "Type a message...",
    "User Attributes": "User Attributes",
    "Users": "Users"
  },
```
I count 32 keys for both the new `AccessRules` and the old translations keys.